src/style.css

```diff
@@ -1,3 +1,73 @@
+:root {
+  --active-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.header {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px 0 #0000000d;
+  padding: 0 50px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding-left: 0;
+  margin: 0;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  height: 60px;
+  margin-left: 20px;
+  align-items: center;
+  text-decoration: none;
+  color: #000;
+}
+
+.nav__item {
+  cursor: default;
+  justify-content: center;
+}
+
+.is-active {
+  margin-left: 0;
```

Comment:
```suggestion
```
it's bad practice to reset the margin in this class, because we can make the third link active, but we don't need in the third link margin-left:0